bip-0003.md

```diff
@@ -0,0 +1,694 @@
+date, and description in a Changelog section. The version number is inspired by semantic versioning (MAJOR.MINOR.PATCH).
+The MAJOR version is incremented if changes to the BIP’s Specification are introduced that are incompatible with prior
+versions (which should be rare after a BIP is Complete, and only happen in well-grounded exceptional cases to a BIP that
+is Deployed). The MINOR version is incremented whenever the specification of the BIP is changed or extended in a
+backward-compatible way. The PATCH version is incremented for other changes to the BIP that are noteworthy (bug fixes,
+test vectors, important clarifications, etc.). Version 1.0.0 is used to label the promotion to
+Complete. A Changelog section may be introduced during the Draft phase to record significant changes (using versions 0.x.y). An example of a Changelog section can be seen in
+[BIP 352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki). After a BIP receives a Changelog, the
+Preamble must indicate the latest version in the Version header. The Changelog highlights revisions to BIPs. A single
+BIP shall not recommend more than one variant of an idea at the same time. An attempt at publishing an updated or
+competing variant of an existing BIP must be published as a separate BIP.
+
+### Adoption of proposals
+
+The BIPs repository does not track the sentiment on proposals and does not track the adoption of BIPs beyond whether they
+are in active use or not. It is not intended for BIPs to list additional implementations beyond the reference
+implementation: the BIPs repository is not a signpost where to find implementations.[^OtherImplementations] After a BIP
+is advanced to Complete, it is up to the Bitcoin community to evaluate, adopt, ignore, or reject a BIP. Individual
+Bitcoin projects are encouraged to publish a list of BIPs they implement. A good example of this at the time of writing
+this BIP can be observed in Bitcoin Core’s [doc/bips.md](https://github.com/bitcoin/bitcoin/blob/master/doc/bips.md)
+file.
+
+### Transferring BIP Ownership
+
+It occasionally becomes necessary to transfer ownership of BIPs to new owners. In general, it would be preferable to
+retain the original authors of the transferred BIP, but that is up to the original authors. A good reason to transfer
+ownership is because the original authors no longer have the time or interest in updating it or following through with
+the BIP process, or are unreachable or unresponsive. A bad reason
+to transfer ownership is because someone doesn't agree with the direction of the BIP. The community tries to build
+consensus around a BIP, but if that's not possible, rather than fighting over control, the dissenters should supply a
+competing BIP.
+
+If someone is interested in assuming ownership of a BIP, they should send an email asking to take over, addressed to the
+original authors, the BIP Editors, and the Bitcoin Development Mailing List. If the authors do not respond in a timely
```

Comment:
```suggestion
original authors, the BIP Editors, and the Bitcoin Development Mailing List. If the authors are unreachable or do not respond in a timely
```

Reply:
I see (thanks!) By "unreachable", I wanted to mean that we don't know how to reach them.

Reply:
We should have an email address for every BIP Author, but either way it’s a fine addition